src/ex2/style.css

```diff
@@ -1,117 +1,207 @@
-html {
-  height: 100%;
-}
-body {
-  margin: 0;
-  padding: 0;
-  height: 100%;
+* {
```

Comment:
using the * selector is problematic - in complex projects it will apply on element you never meant it to work on